### PR TITLE
Expose data spec and add data form

### PR DESCRIPTION
**Pull request by sergeysenja1992**

### CODE REVIEW

File: src/main/java/com/icthh/xm/uaa/web/rest/TenantPropertiesResource.java [hunk 1]
```diff
@@ -77,4 +82,19 @@ public UaaValidationVM validate(@RequestBody String uaaYml) {
...
         return tenantPropertiesService.getTenantProps().getPublicSettings();
     }
+
+    @GetMapping(value = "/uaa/properties/data-schema/{roleKey}")
+    @ApiOperation(value = "Get role data schema", response = UserSpec.class)
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Uaa public settings", response = UserSpec.class),
+            @ApiResponse(code = 500, message = "Internal server error")})
+    @Timed
+    @PreAuthorize("hasPermission({'roleKey': #roleKey}, 'UAA.TENANT.PROPERTIES.UPDATE')")
```
Comment (amedvedchuk) [inline src/main/java/com/icthh/xm/uaa/web/rest/TenantPropertiesResource.java:92]: `UAA.TENANT.PROPERTIES.UPDATE` why the name is UPDATE?

Comment (dtsarenko) [inline src/main/java/com/icthh/xm/uaa/web/rest/TenantPropertiesResource.java:92]: permission's name doesn't match action

Comment (sergeysenja1992) [reply src/main/java/com/icthh/xm/uaa/web/rest/TenantPropertiesResource.java]: fixed

Comment (sergeysenja1992) [reply src/main/java/com/icthh/xm/uaa/web/rest/TenantPropertiesResource.java]: fixed

**Review by dtsarenko — APPROVED** [review]

**Review by amedvedchuk — APPROVED** [review]

**Pull request merged**